test/libsolidity/syntaxTests/types/function_types/definition/assign_function_via_contract_name_to_var.sol

```diff
@@ -0,0 +1,11 @@
+contract A {
+    function f() external {}
+}
+
+contract B {
+    function g() external {
+        function() external f = A.f;
```

Comment:
What if `A.f` is pure?

Reply:
In general you still need an address to staticcall... if you don't know at compile time which function the function pointer will be assigned to, or if you don't have the implementation of the function, then you can't inline - and we could only allow this, if we can inline... I think... in any case, I'd postpone thinking about pure functions wrt this for a later time.

Reply:
Ok. Can you at least add a test case for future reference?